scst-scan/integrate-app-scanning.hbs.md

```diff
@@ -0,0 +1,113 @@
+
+To verify scanning integration:
+
+1. Create an ImageVulnerabilityScan either as a standalone or in the context of a Supply Chain:
+
+    - [Create ImageVulnerabilityScan](./integrate-app-scanning.hbs.md#verifying-scanning-without-supply-chain-integration)
+    - [Create ImageVulnerabilityScan in context of Supply Chain Workload](./integrate-app-scanning.hbs.md#verifying-scanning-with-supply-chain-integration)
+
+2. [Retrieve scan results](./integrate-app-scanning.hbs.md#retrieve-scan-results)
+
+### <a id="verifying-integration"></a> Verifying Scanning without Supply Chain Integration
+
+This section describes how to verify scanning and retrieve scan results.
+#### <a id="trigger-observe-scanning"></a> Trigger and Observe Scanning
+
+To verify that you can scan an image using your ImageVulnerabilityScan:
+
+1. Deploy your ImageVulnerabilityScan to the cluster by running:
+
+    ```console
+    kubectl apply -f image-vulnerability-scan.yaml -n DEV-NAMESPACE
+    ```
+   - Where `DEV-NAMESPACE` is the name of the developer namespace you want to use.
+
+2. Child resources are created.
+
+    - View the child PipelineRun, TaskRuns, and pods
+      ```console
+      kubectl get -l imagevulnerabilityscan,pipelinerun,taskrun,pod -n DEV-NAMESPACE
+      ```
+
+3. When the scanning completes, the status is shown. Specify `-o wide` to see the digest of the image scanned and the location of the published results.
+
+    ```console
+    kubectl get imagevulnerabilityscans -n DEV-NAMESPACE -o wide
+
+    NAME                 SCANRESULT                           SCANNEDIMAGE          SUCCEEDED   REASON
+    generic-image-scan   registry/project/scan-results@digest nginx:latest@digest   True        Succeeded
+
+    ```
+
+### <a id="verifying-scanning-with-supply-chain-integration"></a> Verifying Scanning with Supply Chain Integration
+
+1. Create a sample workload with a pre-built image by using the `tanzu apps workload create` command:
+
+  ```console
+  tanzu apps workload create WORKLOAD-NAME \
+    --app APP-NAME \
+    --type TYPE \
+    --image IMAGE \
+    --namespace DEV-NAMESPACE
+  ```
+
+  Where:
+
+  - `WORKLOAD-NAME` is the name you choose for your workload.
+  - `APP-NAME` is the name of your app.
+  - `TYPE` is the type of your app.
+  - `IMAGE` is the container image that contains the app you want to deploy.
+  - `DEV-NAMESPACE` is the name of the developer namespace where scanning occurs.
+
+  **Note**:
+  - For more info on how to use the Tanzu CLI workload creation see [Create a Workload](../cli-plugins/apps/create-workload.hbs.md) documentation.
+  - Specifying the `--image` flag configures the workload to skip the source scanning step by using the specified pre-built image. Currently SCST - Scan 2.0 only performs image scanning. There are specific requirements for pre-built images. For more details see [Configure your workload to use a prebuilt image](../scc/pre-built-image.hbs.md)
```

Comment:
```suggestion
  - Specifying the `--image` flag configures the workload to skip the source scanning step if has been enabled by using the specified pre-built image. Currently SCST - Scan 2.0 only performs image scanning. There are specific requirements for pre-built images. For more details see [Configure your workload to use a prebuilt image](../scc/pre-built-image.hbs.md)
```
On the fence about removing this entirely, but I think the last part if important to capture.

Reply:
Updated to be more specific on the supply chain being selected.